config/locales/en.yml

```diff
@@ -373,11 +373,11 @@ en:
       course_title: Course title
       course_types:
         basic_course_name: Basic Program
-        basic_course_description: This program type will track edits to any articles or mainspace pages by any participants. If automated wiki edits are enabled on the program's home wiki, then an on-wiki version of this program will be automatically created and updated.
+        basic_course_description: By default, this program type will track edits to any articles or mainspace pages by any participants. If automated wiki edits are enabled on the program's home wiki, then an on-wiki version of this program will be automatically created and updated.
```

Comment:
Yeah, I suppose the 'if enabled' language is already enough to tell users that this isn't always on, and it's fine to leave out the additional details of how it could be configured with the new options (eg, disabling some or all of the edits, even though the hope wiki does have automated edits enabled).